app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,64 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include IdvStepConcern
+    include RenderConditionConcern
+
+    before_action :confirm_step_allowed
+
+    check_or_render_not_found -> { self.class.enabled? }
+
+    def show
+      analytics.idv_doc_auth_how_to_verify_visited(**analytics_arguments)
+      @idv_how_to_verify_form = Idv::HowToVerifyForm.new
+    end
+
+    def update
+      result = Idv::HowToVerifyForm.new.submit(how_to_verify_form_params)
+
+      analytics.idv_doc_auth_how_to_verify_submitted(
+        **analytics_arguments.merge(result.to_h),
+      )
+
+      if result.success?
+        if how_to_verify_form_params['selection'] == Idv::HowToVerifyForm::REMOTE
+          redirect_to idv_hybrid_handoff_url
+        else
+          redirect_to idv_document_capture_url
+        end
+      else
+        flash[:error] = result.first_error_message
+        redirect_to idv_how_to_verify_url
+      end
+    end
+
+    def self.step_info
+      Idv::StepInfo.new(
+        key: :how_to_verify,
+        controller: controller_name,
+        next_steps: [:hybrid_handoff, :document_capture],
+        preconditions: ->(idv_session:, user:) do
+          self.enabled?
```

Comment:
This also needs to confirm that the user has completed the agreement step. It needs to be `self.enabled && idv_session.idv_consent_given`